### PR TITLE
add more editors to editors.md

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

File: EDITORS.md [hunk 1]
```diff
@@ -3,8 +3,9 @@ This is a list of editors and what string to use in `script.config` for them.
 #### NOTE: IF USING ATOM, GO INTO IT, SELECT "ATOM" AND SELECT "INSTALL SCRIPT COMMANDS" BEFORE RUNNING THE SCRIPT!
 Any editor, so long as it has a terminal command to open it, will work with this. If you don't want your new project to open in an editor, set the string to `"none"`. By default, it is set to open VSCode.
 ```
-Visual Studio Code (aka VSCode): "code"
-Atom: "atom"
+Visual Studio Code (aka VSCode): "code" (use "code-insiders" if you use VSCode Insiders)
```
Comment (jarodburchill) [inline EDITORS.md:6]: Can you point me to where you found this to be true? I've never used VSCode Insiders.

Comment (Rexogamer) [reply EDITORS.md:6]: From GitHub Desktop (in particular the code they use for opening the editor):
```
case ExternalEditor.VSCode:
      return Path.join(installLocation, 'bin', 'code.cmd') // remember, CMD must 'useShell'
case ExternalEditor.VSCodeInsiders:
      return Path.join(installLocation, 'bin', 'code-insiders.cmd') // remember, CMD must 'useShell

Comment (jarodburchill) [reply EDITORS.md:6]: Nice catch!

Comment (jarodburchill) [reply EDITORS.md:6]: Can you add sublime text aswell? Also maybe PHPStorm and WebStorm

Comment (Rexogamer) [reply EDITORS.md:6]: It seems sublime uses an exe...

Comment (jarodburchill) [reply EDITORS.md:6]: can it be called from the console though?

Comment (Rexogamer) [reply EDITORS.md:6]: Not without making a shortcut yourself it seems.

Comment (Rexogamer) [reply EDITORS.md:6]: We need to add support for opening editors using their EXE it seems

Comment (jarodburchill) [reply EDITORS.md:6]: it appears you can use `subl` as long as you have Sublime set as an environment variable

Comment (Rexogamer) [reply EDITORS.md:6]: true, but we need the user to know this (maybe run the SETX command and then run subl, or get the user to set it)

Comment (jarodburchill) [reply EDITORS.md:6]: get the user to set it.

**Review by jarodburchill — APPROVED** [review]

**Pull request merged**